powerauth-backend-tests/pom.xml

```
@@ -98,4 +129,298 @@
+                    <execution>
+                        <id>clone-powerauth-crypto</id>
+                        <phase>generate-resources</phase>
+                        <goals><goal>checkout</goal></goals>
+                        <configuration>
+                            <connectionUrl>scm:git:https://github.com/wultra/powerauth-crypto.git</connectionUrl>
+                            <checkoutDirectory>${sources.dir}/powerauth-crypto</checkoutDirectory>
+                            <scmVersion>${git.branch}</scmVersion>
+                            <scmVersionType>branch</scmVersionType>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>clone-powerauth-server</id>
+                        <phase>generate-resources</phase>
+                        <goals><goal>checkout</goal></goals>
+                        <configuration>
+                            <connectionUrl>scm:git:https://github.com/wultra/powerauth-server.git</connectionUrl>
+                            <checkoutDirectory>${sources.dir}/powerauth-server</checkoutDirectory>
+                            <scmVersion>${git.branch}</scmVersion>
+                            <scmVersionType>branch</scmVersionType>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>clone-powerauth-restful-integration</id>
+                        <phase>generate-resources</phase>
+                        <goals><goal>checkout</goal></goals>
+                        <configuration>
+                            <connectionUrl>scm:git:https://github.com/wultra/powerauth-restful-integration.git</connectionUrl>
+                            <checkoutDirectory>${sources.dir}/powerauth-restful-integration</checkoutDirectory>
+                            <scmVersion>${git.branch}</scmVersion>
+                            <scmVersionType>branch</scmVersionType>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>clone-powerauth-cmd-tool</id>
+                        <phase>generate-resources</phase>
+                        <goals><goal>checkout</goal></goals>
+                        <configuration>
+                            <connectionUrl>scm:git:https://github.com/wultra/powerauth-cmd-tool.git</connectionUrl>
+                            <checkoutDirectory>${sources.dir}/powerauth-cmd-tool</checkoutDirectory>
+                            <scmVersion>${git.branch}</scmVersion>
+                            <scmVersionType>branch</scmVersionType>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>clone-enrollment-server-wultra</id>
+                        <phase>generate-resources</phase>
+                        <goals><goal>checkout</goal></goals>
```

Nitpick: Is the formatting compact on purpose?
```suggestion
                        <goals>
                            <goal>checkout</goal>
                        </goals>
```